### PR TITLE
add setIsRpcDown/isRpcDown

### PR DESCRIPTION
**Pull request by andrewdamelio**
https://github.com/AugurProject/turbo/issues/780

### CODE REVIEW

File: packages/simplified/src/modules/market/trading-form.tsx [hunk 1]
```diff
@@ -302,6 +305,11 @@ const TradingForm = ({ initialSelectedOutcome, marketType = YES_NO, amm }: Tradi
       .catch((error) => {
         setWaitingToSign(false);
         console.log("Error when trying to trade: ", error?.message);
+        if (error?.message) {
```
Comment (bthaile) [inline packages/simplified/src/modules/market/trading-form.tsx]: This only catches if user is making a trade. There are other situations where rpc being down effects the user. 

Example when user isn't logged in and rpc endpoint is down, markets don't get loaded or don't get updated. 
Also if user is trying to add liquidity. 

Need to catch these situations, look at adding this catch in data store. 
`packages/comps/src/stores/data.tsx` when `getMarketInfos` is called, that will catch the majority of issues with rpc.

**Review by bthaile — CHANGES_REQUESTED** [review]
see comment

**Comment by andrewdamelio** [comment]
@bthaile will fix!

**Review by bthaile — APPROVED** [review]
approved

**Pull request merged**